File: Java/Account.java
package Java;

public class Account 
{
    // Atributes 

    private Integer id;
    private String name;
    private String document;
    private String email;
    private String password;

    // Constructor

    public Account(String name, String document)
    {
        this.name = name;
        this.document = document;
    }

    // Getters & Setters

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDocument() {
        return document;
    }

    public void setDocument(String document) {
        this.document = document;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    // Methods
}

File: PHP/Account.php
<?php

class Account
{

    // Atributes

    private $id;
    private $name;
    private $document;
    private $email;
    private $password;
    
    // Constructor
    
    public function __construct($name, $document)
    {
        $this->name     = $name;
        $this->document = $document;
    }

    // Getters and Setters 
    
    public function getId()
    {
        return $this->id;
    }

    public function setId($id)
    {
        $this->id = $id;
    }

    public function getName()
    {
        return $this->name;
    }

    public function setName($name)
    {
        $this->name = $name;
    }

    public function getDocument()
    {
        return $this->document;
    }

    public function setDocument($document)
    {
        $this->document = $document;
    }

    public function getEmail()
    {
        return $this->email;
    }

    public function setEmail($email)
    {
        $this->email = $email;
    }

    public function getPassword()
    {
        return $this->password;
    }

    public function setPassword($password)
    {
        $this->password = $password;
    }

    // Methods

    
}
